packages/lib/isOutOfBounds.tsx

```diff
@@ -263,6 +272,17 @@ export function isTimeViolatingFutureLimit({
       startOfRangeStartDayInEventTz: periodLimits.startOfRangeStartDayInEventTz.format(),
       endOfRangeEndDayInEventTz: periodLimits.endOfRangeEndDayInEventTz.format(),
     });
+    if (isBeforeRangeStart || isAfterRangeEnd)
+      log.warn(
+        "Booking is out of bounds due to range start and end.",
+        safeStringify({
+          formattedDate: dateInSystemTz.format(),
+          isBeforeRangeStart,
+          isAfterRangeEnd,
+          startOfRangeStartDayInEventTz: periodLimits.startOfRangeStartDayInEventTz.format(),
+          endOfRangeEndDayInEventTz: periodLimits.endOfRangeEndDayInEventTz.format(),
+        })
+      );
```

Comment:
We log all necessary details when user attempts to book outside the period range